views/partials/search-form-fields.html

```diff
@@ -183,7 +183,7 @@
   <div class="controls">
     <button type="submit"><span>Search</span></button>
     {% if advancedSearchIsVisible %}
```

Comment:
This line shouldn't be necessary, since the whole container is invisible if `advancedSearchIsVisible` is false. Also, it doesn't show up on my machine the way it's built right now.